net/sockstats/sockstats_tsgo.go

```diff
@@ -336,6 +336,10 @@ func (rm *radioMonitor) radioHighPercent() int64 {
 	}
 	periodStart := now - periodLength // start of current reporting period
 
+	if periodLength <= 0 {
```

Comment:
Nit: could move this to be inside the if block at line 335 (and check `t <= 0`), since that's the only place where it can actually be 0.

Reply:
done